provider/provider_yaml_test.go

```diff
@@ -142,12 +143,12 @@ func TestNoGlobalProjectWarning(t *testing.T) {
 	t.Setenv("GOOGLE_PROJECT", "")
 
 	test := pulumitest.NewPulumiTest(t, "test-programs/project-bucket",
-	opttest.LocalProviderPath(providerName, filepath.Join(cwd, "..", "bin")))
+		opttest.LocalProviderPath(providerName, filepath.Join(cwd, "..", "bin")))
 
 	test.SetConfig("gcpProj", proj)
 	res := test.Up()
 	require.Contains(
-		t, res.StdOut, 
```

Comment:
Was this not linted?

Reply:
The linter was off. I opened https://github.com/pulumi/pulumi-gcp/pull/1699 to fix it.